ros_buildfarm/templates/__init__.py

```diff
@@ -72,14 +75,18 @@ def _escape_value(value):
 
 
 def _expand_snippet(snippet_name, **kwargs):
+    """Expand snippet using given snippet_name"""
     template_name = 'snippet/%s.xml.em' % snippet_name
     _expand_template(template_name, **kwargs)
 
 
 def _expand_template(template_name, **kwargs):
-    global template_basepath
+    """Expand template call for interpreter"""
+    template_name = os.path.join('templates', template_name)
+    template_package = _find_first_template(template_name, **kwargs)
+
     global interpreter
-    template_path = os.path.join(template_basepath, template_name)
+    template_path = pkg_resources.resource_filename(template_package, template_name)
```

Comment:
Doesn't relying on `pkg_resources` to locate the templates break using the package from source? If yes, I would rather not want to sacrifice that.